app/models/book.rb

```diff
@@ -0,0 +1,2 @@
+class Book < ApplicationRecord
+end
```

Comment:
you need to validate all the fields that you declare as `required` in the migrations, if you don't do this Rails could say that certain object is valid (for example a Book without a genre) and when it sends it to the database to save it, it is going to fail and raise and Exception. You can try this by doing `Book.create` in your rails console